### PR TITLE
Return row object rather than ID on bulk action

### PR DESCRIPTION
**Pull request by dextermb**

### CODE REVIEW

File: src/DynamicDataTable.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ class DynamicDataTable extends Component {
         super(props);
 
         this.state = {
-            checkedRowIds: [],
+            checkRows: [],
```
Comment (DivineOmega) [inline src/DynamicDataTable.jsx]: Should this be `checkedRows`?

File: src/DynamicDataTable.jsx [hunk 2]
```diff
@@ -280,48 +280,73 @@ class DynamicDataTable extends Component {
...
-    checkboxChange(e) {
-        const target = e.target;
-        const props = this.props;
-        const rows = props.rows;
+    checkboxChange({ event, row }) {
+        const { rows } = this.props;
+        const { target } = event;
 
-        if (target.value === 'all') {
+        if (row === 'all') {
```
Comment (DivineOmega) [inline src/DynamicDataTable.jsx]: Does this check still work?

Comment (dextermb) [reply src/DynamicDataTable.jsx]: Yup

Comment (DivineOmega) [reply src/DynamicDataTable.jsx]: Magic. ✨

File: src/DynamicDataTable.jsx (hunk 1 above)
Comment (dextermb) [reply src/DynamicDataTable.jsx]: No, in the state it is `checkRows`.

**Pull request merged**